src/commands/libcoreReset.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import { createCommand, Command } from 'helpers/ipc'
+import { of } from 'rxjs'
+import withLibcore from 'helpers/withLibcore'
+
+type Input = void
+type Result = boolean
+
+const cmd: Command<Input, Result> = createCommand('libcoreReset', () => {
+  withLibcore(core => core.getPoolInstance().freshResetAll())
```

Comment:
return `fromPromise(withLibcore(...`

Reply:
safer as you commented ;)

Reply:
Cool! ![](https://phaazon.net/media/uploads/very_nice.gif)